http/server.go

```diff
@@ -300,6 +301,7 @@ func (h *handler) ChainInfo(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Cache-Control", "public, max-age=604800, immutable")
 	w.Header().Set("Expires", time.Now().Add(7*24*time.Hour).Format(http.TimeFormat))
 	w.Header().Set("Content-Type", "application/json")
+	w.Header().Set("Digest", "drand="+hex.EncodeToString(info.Hash()))
```

Comment:
This should probably be `X-Digest` as it isn't standard header.

Reply:
it is a standard header, just not a standard algorithm 😁  Maybe "x-drand"?

Reply:
ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Digest

Reply:
Hmm, right, maybe `x-info-hash`